profiler/include/profiler/profile_contraction_impl.hpp

```diff
@@ -168,37 +190,37 @@ int profile_contraction_impl(int do_verification,
                 static_cast<DataType*>(a_device_buf.GetDeviceBuffer()),
                 static_cast<DataType*>(b_device_buf.GetDeviceBuffer()),
                 std::array<const void*, 1>{d_device_buf.GetDeviceBuffer()},
-                static_cast<DataType*>(c_device_buf.GetDeviceBuffer()),
-                a_ms_ks_lengths,
+                static_cast<DataType*>(e_device_buf.GetDeviceBuffer()),
+                a_m_k_lengths,
```

Comment:
The `s` in variable means we deal with multiple Ms and multiple Ns.

Reply:
Restored.